docs/pages/using_merge_receiver.md

```diff
@@ -0,0 +1,345 @@
+
+This callback should be processed quickly, since it will interfere with the receipt and processing
+of other sACN packets on the universe.
+
+<!-- CODE_BLOCK_START -->
+```c
+void my_universe_data_callback(sacn_merge_receiver_t handle, uint16_t universe, const uint8_t* slots, 
+                               const sacn_source_id_t* slot_owners, void* context)
+{
+  // Check handle and/or context as necessary...
+
+  // You wouldn't normally print a message on each sACN update, but this is just to demonstrate the
+  // fields available:
+  printf("Got new merge results on universe %u\n", universe);
+
+  // Example for an sACN-enabled fixture...
+  bool my_slots_all_sourced = true;
+  for(unsigned int i = 0; i < MY_DMX_FOOTPRINT; ++i)
+    my_slots_all_sourced = my_slots_all_sourced && SACN_DMX_MERGER_SOURCE_IS_VALID(slot_owners, my_start_addr + i);
+
+  if(my_slots_all_sourced)
+  {
+    memcpy(my_data_buf, &slots[my_start_addr], MY_DMX_FOOTPRINT);
+    // Act on the data somehow
+  }
+}
+```
+<!-- CODE_BLOCK_MID -->
+```cpp
+void MyNotifyHandler::HandleMergedData(uint16_t universe, const uint8_t* slots, const sacn_source_id_t* slot_owners)
+{
+  // You wouldn't normally print a message on each sACN update, but this is just to demonstrate the
+  // fields available:
+  std::cout << "Got new merge results on universe " << universe << "\n";
+
+  // Example for an sACN-enabled fixture...
+  bool my_slots_all_sourced = true;
+  for(unsigned int i = 0; i < MY_DMX_FOOTPRINT; ++i)
+    my_slots_all_sourced = my_slots_all_sourced && SACN_DMX_MERGER_SOURCE_IS_VALID(slot_owners, my_start_addr + i);
+
+  if(my_slots_all_sourced)
+  {
+    memcpy(my_data_buf, &slots[my_start_addr], MY_DMX_FOOTPRINT);
+    // Act on the data somehow
+  }
+}
+```
+<!-- CODE_BLOCK_END -->
+
+If non-NULL start code, non-PAP sACN data is received, it is passed through directly to the
+non-DMX data callback.
+
+<!-- CODE_BLOCK_START -->
+```c
+void my_universe_non_dmx_callback(sacn_merge_receiver_t handle, uint16_t universe, const EtcPalSockAddr* source_addr, 
+                                  const SacnHeaderData* header, const uint8_t* pdata, void* context)
+{
+  // Check handle and/or context as necessary...
+
+  // You wouldn't normally print a message on each sACN update, but this is just to demonstrate the
+  // header fields available:
+  char addr_str[ETCPAL_IP_STRING_BYTES];
+  etcpal_ip_to_string(&source_addr->ip, addr_str);
+
+  char cid_str[ETCPAL_UUID_STRING_BYTES];
+  etcpal_uuid_to_string(&header->cid, cid_str);
+
+  printf("Got non-DMX sACN update from source %s (address %s:%u, name %s) on universe %u, priority %u, start code %u\n",
+         cid_str, addr_str, source_addr->port, header->source_name, universe, header->priority, header->start_code);
+
+  // Example for an sACN-enabled fixture...
+  if (header->start_code == MY_CUSTOM_START_CODE && my_start_addr + MY_DMX_FOOTPRINT <= header->slot_count)
```

Comment:
Non-DMX data wouldn't have a start address or a DMX footprint (those are DMX concepts). We should just replace this block with the `// Act on the data somehow` comment.